incubator/zookeeper/templates/pdb.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: "{{ printf "pd-%s" .Release.Name | trunc 24 }}"
+  annotations:
+    helm.sh/created: {{.Release.Time.Seconds | quote }}
+  labels:
+    heritage: {{.Release.Service | quote }}
+    release: {{.Release.Name | quote }}
+    chart: "{{.Chart.Name}}-{{.Chart.Version}}"
+    component: "{{.Release.Name}}"
+spec:
+  selector:
+    matchLabels:
+      component: "{{.Release.Name}}"
+  minAvailable: {{.Values.MinAvailable}}
```

Comment:
This should be computed from `{{.Values.Servers}}` such the MinAvailable provides quorum across the cluster.